packages/frontend/src/pages/positions/PositionDetails.tsx

```diff
@@ -1,35 +1,69 @@
+import { AssetId } from '@explorer/types'
 import React from 'react'
 
 import { AssetNameCell } from '../common/AssetNameCell'
 import { Page } from '../common/Page'
 import { formatTimestamp, PageHeaderStats } from '../common/PageHeaderStats'
 import { SimpleLink } from '../common/SimpleLink'
-import { Table } from '../common/Table'
+import { Column, Table } from '../common/Table'
 import { formatHash } from '../formatHash'
 import { formatLargeNumber } from '../formatLargeNumber'
 import { formatTime } from '../formatTime'
 import { formatUSDCents } from '../formatUSDCents'
 import { PositionDetailsProps } from './PositionDetailsProps'
 
-const balanceTableColumns = [
-  { header: 'Name' },
-  { header: 'Balance', numeric: true },
-  { header: 'Unit price', numeric: true },
-  { header: 'Value', numeric: true },
-]
-const buildBalanceTableRow = ({
+const balanceTableColumns = (ownedByYou: boolean) => {
+  const columns: Column[] = [
+    { header: 'Name' },
+    { header: 'Balance', numeric: true },
+    { header: 'Unit price', numeric: true },
+    { header: 'Value', numeric: true },
+  ]
+
+  if (ownedByYou) {
+    columns.push({ header: 'Forced', maxWidthClass: 'w-0' })
+  }
+
+  return columns
+}
+
+const ActionButton = ({ text }: { text?: string }) => (
+  <>{text && <button className="px-3 rounded bg-blue-100">{text}</button>}</>
+)
+
+const actionButtonText = ({
   assetId,
   balance,
-  totalUSDCents,
-  price,
-}: PositionDetailsProps['assets'][number]) => ({
-  cells: [
-    <AssetNameCell assetId={assetId} />,
-    balance.toString(),
-    price ? `${formatUSDCents(price)}` : '-',
-    formatUSDCents(totalUSDCents),
-  ],
-})
+}: Pick<PositionDetailsProps['assets'][number], 'assetId' | 'balance'>) => {
```

Comment:
why not just `string`

Reply:
This is not the return value, but arguments. Simplyfied it a little bit for sake of readability